### DISCUSSION TITLE
How's the project doing?

### DISCUSSION THREAD

**Discussion by insanpraja** [tensorbase/tensorbase#349] [General] [answered]
Are you guys moving somewhere else or something? hope you all doing fine! promising project btw.

**Comment by jinmingjian** [comment, marked as answer]
Hi, @insanpraja really thanks for your interests and querying. 

We are still fine:)  In the end of 2022,  we released a new product,  called [JoinBase](https://joinbase.io/), and you can download the initial binary distribution from the link. The interesting side of JoinBase is, it solved one common pitfall of opensource OLAP DB: the high latency of queries.  In the core of JoinBase, we rebuild the full end-to-end pipeline to make that the latency of a REST SQL call is so low that the concurrency of server is highest.  I often call JoinBase is "3/4 HTAP in one engine".  Of course, the JoinBase has other interesting goods for example, one binary, IoT protocol, JSON and arbitrary message processing extensions. It is great that all these are done in a final binary <5MB. The download-able binary is ~9MB, because some AI runtime was integrated with it. Welcome to play with it yourself.

Currently we are moved to a new database product, to answer the coming of the era of ChatGPT. The most part of JoinBase will be moved in the new product. We will announce this in some time soon.

For TensorBase (TB, for short), I am considering restart it. TB belongs to the dual era of AI and data. And it is the time. In the initial version of TB, in fact, the AI method  has been used to solve the DB problem. This is the source of the TB name. But after 3 years (from the born of TB) gone, I see few thing more than "copy and spawn"...

For new TB, it may be like DuckDB in Rust but with some great extensions. I still need many time on the current new product. It is estimated to take several months to restart TB (and release new open source projects as well).

**Reply by insanpraja** [reply]
Glad to hear that you guys are still thriving. I hope that TB community continue to support this great open source ecosystem. DuckDB approach seems wonderful, we have some internal apps with DuckDB, and it suffer when dealing with large data. I hope TB could tackle this.

Cheers!
